Headers/Foundation/NSKeyValueObserving.h

```diff
@@ -229,7 +229,7 @@ typedef NSString *NSKeyValueChangeKey;
  * they should also be sent for dependentKey.
  */
 + (void) setKeys: (NSArray*)triggerKeys
-triggerChangeNotificationsForDependentKey: (NSString*)dependentKey;
+triggerChangeNotificationsForDependentKey: (NSString*)dependentKey __attribute__((deprecated));
```

Comment:
@rfm how can I mark a function as deprecated without breaking GCC < 3.1 choking on this header? Do we have a macro for that? `OBJC_DEPRECATED` is always a no-op.